R/comparePlot.R

```diff
@@ -134,6 +125,8 @@ comparePlot <- function(compareDF,
 
     assertthat::assert_that(sum(apply(compareDF, 2, FUN = is.numeric)) >= 2,
                             msg = "Need at least two numeric columns in compareDF.")
+    assertthat::assert_that(plotType %in% c("ggplot", "canvasxpress"),
```

Comment:
utilize "canvasXpress" like the package name case.

Reply:
ok